packages/flutter/test/gestures/velocity_tracker_test.dart

```diff
@@ -144,4 +144,22 @@ void main() {
       }
     }
   });
+
+  test('Assume zero velocity when there are no recent samples', () {
```

Comment:
This method needs to be async, which may fix all/most of the CI errors.

```suggestion
  test('Assume zero velocity when there are no recent samples', () async {
```

Reply:
thanks for taking a look at this! 

i'll push an update! (later tonight)